demo/variable_definitions/read_variable_definitions.ipynb

```diff
@@ -0,0 +1,375 @@
+    "        \"nb\": \"For personer født i utlandet, er dette (med noen få unntak) eget fødeland. For personer født i Norge er det foreldrenes fødeland. I de tilfeller der foreldrene har ulikt fødeland, er det morens fødeland som blir valgt. Hvis ikke personen selv eller noen av foreldrene er utenlandsfødt, hentes landbakgrunn fra de første utenlandsfødte en treffer på i rekkefølgen mormor, morfar, farmor eller farfar.\",\n",
+    "        \"nn\": \"For personar fødd i utlandet, er dette (med nokre få unntak) eige fødeland. For personar fødd i Noreg er det fødelandet til foreldra. I dei tilfella der foreldra har ulikt fødeland, er det fødelandet til mora som blir valt. Viss ikkje personen sjølv eller nokon av foreldra er utenlandsfødt, blir henta landsbakgrunn frå dei første utenlandsfødte ein treffar på i rekkjefølgja mormor, morfar, farmor eller farfar.\",\n",
+    "    },\n",
+    "    \"classification_reference\": \"91\",\n",
+    "    \"unit_types\": [\"01\", \"02\"],\n",
+    "    \"subject_fields\": [\"he04\"],\n",
+    "    \"contains_sensitive_personal_information\": True,\n",
+    "    \"measurement_type\": None,\n",
+    "    \"valid_from\": \"2003-01-01\",\n",
+    "    \"external_reference_uri\": \"https://www.ssb.no/a/metadata/conceptvariable/vardok/1919/nb\",\n",
+    "    \"comment\": {\n",
+    "        \"nb\": \"Fra og med 1.1.2003 ble definisjon endret til også å trekke inn besteforeldrenes fødeland.\",\n",
+    "        \"nn\": \"Fra og med 1.1.2003 ble definisjon endret til også å trekke inn besteforeldrenes fødeland.\",\n",
+    "        \"en\": \"As of 1 January 2003, the definition was changed to also include the grandparents' country of birth.\",\n",
+    "    },\n",
+    "    \"related_variable_definition_uris\": [\"https://example.com/\"],\n",
+    "    \"contact\": {\n",
+    "        \"title\": {\n",
+    "            \"en\": \"Division for population statistics\",\n",
+    "            \"nb\": \"Seksjon for befolkningsstatistikk\",\n",
+    "            \"nn\": \"Seksjon for befolkningsstatistikk\",\n",
+    "        },\n",
+    "        \"email\": \"s320@ssb.no\",\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "print_json(data=vardef_request(\"/variable-definitions\", \"POST\", body))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Edit a draft"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "body = {\n",
```

Comment:
Maybe add one comment here about how to update status if the user would like to test operations for published variable definitions

Reply:
I believe there already are published definitions available to use.

Reply:
Yes, but only for certain variable owners, or will people using this demo belong to one of those groups?

Reply:
I understand now that all SSB employees are members of "dapla-felles-developers" and will have access to published variables 😊